dist/Dockerfile

```diff
@@ -1,18 +1,18 @@
 # Dockerfile
 
-FROM  java:8
+FROM anapsix/alpine-java:8_server-jre
```

Comment:
Why use this image as a base instead of one of the officially supported openjdk alpine base images?

Reply:
also keep in mind that if you decide to use ssl in the tcp connection the JDK implementation generates a lot of garbage and there are no alpine artifacts for netty-tcnative.

Reply:
@kuujo try `FROM openjdk/8-jdk-slim` compressed size is 91 M.

https://hub.docker.com/r/library/openjdk/tags/

Reply:
@kuujo try `FROM openjdk/8-jdk-alpine` it's an official image.